draft-ietf-quic-multipath.md

```diff
@@ -616,7 +592,36 @@ be controlled by the combination of one or several of the following:
    uses a series of consecutive sequence numbers without creating
    holes.
 
-### ACK Delay Considerations
+### NULL-CID-Loss-and-Congestion
```

Comment:
```suggestion
### Congestion Control, Loss Recovery and NULL CID Considerations
```

Reply:
Let's first apply @mirjak's suggestion to replace NULL by zero-length...

Reply:
The new title is "Loss and Congestion Handling With Zero-Length CID". I hope it works.